lib/_tls_wrap.js

```diff
@@ -1570,9 +1568,7 @@ Server.prototype.setOptions = deprecate(function(options) {
   if (options.sessionIdContext) {
     this.sessionIdContext = options.sessionIdContext;
   } else {
-    this.sessionIdContext = crypto.createHash('sha1')
-      .update(process.argv.join(' '))
-      .digest('hex')
+    this.sessionIdContext = crypto.hash('sha1', process.argv.join(' '), 'hex')
```

Comment:
Why did you drop the `slice(0, 32)` off the end? I believe this will return a string with 40 hex digits while the original was truncated to 32.

Reply:
I didn't, it's still there :-))

Reply:
ha! totally misread the diff lol.. I'm going to blame the github UI... not because it wasn't obvious but because I cannot admit that I simply misread the thing I simply misread.